src/distributions/microsoft/installer.ts

```diff
@@ -40,6 +40,11 @@ export class MicrosoftDistributions extends JavaBase {
     if (this.architecture !== 'x64' && this.architecture !== 'aarch64') {
       throw new Error(`Unsupported architecture: ${this.architecture}`);
     }
+
+    if (!this.stable) {
+      throw new Error('Unstable versions are not supported');
```

Comment:
What's the value of adding a special error message here?  We only have two hardcoded versions currently.  If someone requests something that's not one of those versions, we'll raise an error and show the available versions (lines 62-70 of this file).

Reply:
```suggestion
      throw new Error('Early access versions are not supported');
```

While it's true that it's not a stable version, "unstable" has a stronger denotation.